akka-docs/src/main/paradox/typed/failure-detector.md

```diff
@@ -0,0 +1,70 @@
+# Phi Accrual Failure Detector
+
+Remote Death Watch uses heartbeat messages and the failure detector to 
+
+* Detect network failures and JVM crashes
+* Generate the `Terminated` message to the watching actor on failure 
+* Graceful termination of watched actors  
```

Comment:
It doesn't gracefully terminate watched actors, does it?

Reply:
Here's how `graceful termination of watched actors ` is currently used
```
// remote
Under the hood remote death watch uses heartbeat messages and a failure detector to generate Terminated message from network failures and JVM crashes, in addition to graceful termination of watched actor.

// cluster
Death watch uses the cluster failure detector for nodes in the cluster, i.e. it detects network failures and JVM crashes, in addition to graceful termination of watched actor. 
 
For example in:
```
* https://doc.akka.io/docs/akka/current/remoting-artery.html#failure-detector
* https://doc.akka.io/docs/akka/current/remoting.html#failure-detector
* https://doc.akka.io/docs/akka/current/cluster-usage.html#worker-dial-in-example
* https://doc.akka.io/docs/akka/current/cluster-usage.html#failure-detector

What would be prefer it to say I guess

Reply:
It seems pretty clear that is what we're saying, no?

```
Remote DeathWatch uses heartbeat messages and the failure detector to 

* detect network failures and JVM crashes
* generate the `Terminated` message to the watching actor on failure 
* gracefully terminate of watched actors   (<- Remote DeathWatch, using FD to...)
```

Reply:
I still think it is weird, it could crash an actor that is watching a remote actor, if it doesn't receive the terminated, but crashing not very graceful and is not happening to the watched actor. 

Unless you use SBR (or auto-down) no downing happens, and with just remoting the remote node will be quarantined and, but that doesn't do anything with the watched actor either (which is why quarantining needs an actor system restart)

Reply:
@patriknw is there something here I'm missing?

Reply:
We could see the issue as out of scope, for a later change since this ticket is about the consolidation of existing content: classic -> typed swap, vs updating content.
WDYT?

Reply:
Sure, created https://github.com/akka/akka/issues/27751 to track this particular weirdness.